routes/user.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const router = express.Router();
+const ragiController = require("../controllers/user");
+const checkExist =require("../middleware/varify")

```

Comment:
correct the spelling of verify?
can you please add spell checker vs code extension for that

Reply:
done